Common/Orders/OrderEvent.cs

```diff
@@ -123,6 +152,15 @@ public override string ToString()
                 ? string.Format("OrderID: {0} Symbol: {1} Status: {2}", OrderId, Symbol, Status) 
                 : string.Format("OrderID: {0} Symbol: {1} Status: {2} Quantity: {3} FillPrice: {4}", OrderId, Symbol, Status, FillQuantity, FillPrice);
         }
+
+        /// <summary>
+        /// Returns a clone of the current object.
+        /// </summary>
+        /// <returns>The new clone object</returns>
+        public OrderEvent Clone()
+        {
+            return (OrderEvent)MemberwiseClone();
```

Comment:
This should be safe given the only reference type `Symbol` is immutable. Also, this is certainly not performance sensitive.